app/depict/src/main/java/org/openscience/cdk/depict/FreeHepWrapper.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright (c) 2015 John May <jwmay@users.sf.net>
+ *
+ * Contact: cdk-devel@lists.sourceforge.net
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or (at
+ * your option) any later version. All we ask is that proper credit is given
+ * for our work, which includes - but is not limited to - adding the above
+ * copyright notice to the beginning of your source code files, and to any
+ * copyright notice that you may distribute with programs based on this work.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public
+ * License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 U
+ */
+
+package org.openscience.cdk.depict;
+
+import org.freehep.graphicsio.pdf.PDFGraphics2D;
+import org.freehep.graphicsio.svg.SVGGraphics2D;
+
+import java.awt.Dimension;
+import java.awt.Graphics2D;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.nio.charset.StandardCharsets;
+import java.util.Properties;
+
+/**
+ * Internal - wrapper around the FreeHEP vector graphics output that makes things consistent
+ * in terms of writing the required headers and footers.
+ */
+final class FreeHepWrapper {
+
+    private final ByteArrayOutputStream bout;
+    private final String                fmt;
+    final         Graphics2D            g2;
+
+    public FreeHepWrapper(String fmt, double w, double h) {
+        try {
+            this.g2 = createGraphics2d(this.fmt = fmt,
+                                       this.bout = new ByteArrayOutputStream(),
+                                       new Dimension((int) Math.ceil(w), (int) Math.ceil(h)));
+        } catch (IOException e) {
+            throw new InstantiationError("Could not create Vector Graphics output:" + e.getMessage());
+        }
+    }
+
+    private static Graphics2D createGraphics2d(String fmt, OutputStream out, Dimension dim) throws IOException {
+        switch (fmt) {
+            case Depiction.SVG_FMT:
+                SVGGraphics2D svg = new SVGGraphics2D(out, dim);
+                svg.setCreator("Chemistry Development Kit (CDK)");
```

Comment:
Or "Chemistry Development Kit (http://cdk.sf.net/)"... ?

A long time ago I read something about Google indexing SVG image content ... not sure, though; would have to check...